### PR TITLE
Updating Planter to use Bazel 0.21.0

### PR DESCRIPTION
**Pull request by chrislovecnm**
- Updated Makefile version number
- Updated planter.sh version number

### CODE REVIEW

**Comment by chrislovecnm** [comment]
cc @milesmatthias

File: planter/planter.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ set -o nounset
 # these can be overridden but otherwise default to the current stable image
 # used to build kubernetes from the master branch
 IMAGE_NAME="${IMAGE_NAME:-gcr.io/k8s-testimages/planter}"
-TAG="${TAG:-0.19.2}"
+TAG="${TAG:-0.20.0}"
```
[the remark this answers is not in the document]
Comment (BenTheElder) [reply planter/planter.sh]: I have pushed the image though, FWIW

[the remark this answers is not in the document]
Comment (chrislovecnm) [reply planter/planter.sh]: fixed

**Comment by chrislovecnm** [comment]
@milesmatthias the planter container has been pushed to the repo

**Comment by chrislovecnm** [comment]
@BenTheElder thanks as ALWAYS!!!!

File: planter/planter.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (BenTheElder) [reply planter/planter.sh]: thanks

**Pull request merged**